packages/mediacenter/kodi/scripts/ffmpeg

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
+b="/storage/.kodi/addons/tools.ffmpeg-tools/bin/ffmpeg"
+if [ -f "$b" ]; then
```

Comment:
It should be check for executable (-x) ?

Reply:
yes makes more sens, tx